src/web/mjs/connectors/MangaRaw.mjs

```diff
@@ -0,0 +1,78 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class MangaRaw extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mangaraw';
+        super.label = 'MangaRaw (Manga Raw Club)';
+        this.tags = ['multi-lingual', 'manga'];
+        this.url = 'https://www.manga-raw.club';
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+        let request = new Request(this.url +'/browse/', this.requestOptions);
+        let data = await this.fetchDOM(request, '#explore > nav.paging a:nth-last-of-type(2)');
+        let pageCount = parseInt(data[0].href.match(/results=(\d+)&/)[1]);
+        for(let page = 1; page <= pageCount; page++) {
+            let mangas = await this._getMangasFromPage(page);
+            mangaList.push(...mangas);
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        let request = new Request(`${this.url}/browse/?results=${page}`, this.requestOptions);
+        let data = await this.fetchDOM(request, '.novel-item a',3);
+        await this.wait(250);
```

Comment:
Is this wait required?
May move the wait into the `for` loop of `_getMangas`

Reply:
yea , it got a 40x error without it (i think it was 403 i don't remember)